panels/site_sections/schedule.py

```diff
@@ -244,6 +244,7 @@ def panelists_owed_refunds(self, session):
                           if a.paid == c.HAS_PAID or a.paid == c.PAID_BY_GROUP and a.group and a.group.amount_paid]
         }
 
+    @unrestricted
```

Comment:
This is an unrelated fix; I realized that I forgot to make the page that we will eventually email out to our panelists showing their schedule unrestricted, which would have prevented them from being able to see it!